Feature article for IPv6 (#1820)
linkerd/website: linkerd.io/content/2.16/features/ipv6.md

really an issue.

The real benefits of IPv6 in this context deal more with things like increased
security, improved performance and generally a more modern stack that the
cluster network can benefit from.

Dual-stack networking on the other hand allows declaring services backed by both
IPv4 and IPv6 EndpointSlices (note that that the legacy Endpoints resources
don't support dual-stack). This becomes useful when you want to migrate your
services to IPv6 gradually, and thus require a mixed environment during the
transition.

## Kubernetes Providers Support

As of this writing these are the options provided by the major cloud providers
(included k3d and kind for local testing / CI):

{{< table >}}
| Provider | Dual-Stack Supported | IPv6-only Supported |
|----------|----------------------|---------------------|
|    AKS   |  :white_check_mark:  |        :x:          |
|    EKS   |         :x:          | :white_check_mark:  |
|    GKE   |  :white_check_mark:  |        :x:          |
|    Kind  |  :white_check_mark:  | :white_check_mark:  |
|    k3d   |         :x:          |        :x:          |
{{< /table >}} 

> 

Reviewer: Thank you for writing these, but I suggest removing both these sections. We don't really need to advocate for IPv6 as a technology, and I don't want to maintain a compatibility table for cloud provider support.

## Linkerd IPv6 Configuration

IPv6 support in Linkerd is disabled by default. To enable, just set

> `proxy.disableIPv6=false`. This setting is to be applied to the control plane
> and the linkerd-cni plugin, if used.

Reviewer 2: ```suggestion
`proxy.disableIPv6=false` when installing the control plane and, if you use it, the linkerd-cni plugin.
```

??

This mode doesn't make any difference in how Linkerd works. Bear in mind however
that when faced with a dual-stack service, the proxy will only forward traffic

> to the endpoints in the IPv6 EndpointSlice.

Reviewer: Enabling IPv6 support does not generally change how Linkerd operates, except in one way: when enabled on a dual-stack cluster, Linkerd will only use the IPv6 endpoints of destinations and will not use the IPv4 endpoints.
